core/cas-server-core-web/src/main/java/org/apereo/cas/web/SimpleUrlValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.apereo.cas.web;
+
+import org.apache.commons.validator.routines.UrlValidator;
+
+/**
+ * Implementation of CAS {@link org.apereo.cas.web.UrlValidator}
+ * which delegates requests to a {@link #urlValidator} instance.
+ * 
+ * @author swoeste
+ * @since 5.2.0
+ */
+public class SimpleUrlValidator implements org.apereo.cas.web.UrlValidator {
```

Comment:
Perhaps an opportunity to create a test case for this class and the factory as well?